### PR TITLE
Various Fixes

### PR DESCRIPTION
**Pull request by redhatrises**
- Update dockerfile to reduce build steps
- Run autopep8 to fix some pep8 complaints
- Move the dockerfile to the top level

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,7 @@
+FROM python:3.7-slim
```
Comment (shawndwells) [inline Dockerfile:1]: Planning ahead, does this use a base image that our friends in regulated environments can use? Eg FIPS capable.

File: EventStreams/src/chronicleclient.py [hunk 2]
```diff
@@ -70,17 +73,19 @@ def handle_exit(self, code, message):
...
-            attempt = 0      
-            headers = {'Content-Type':'application/json'}
-            payload = {"events":[mapper(response)]}
-            r = request("POST", "https://malachiteingestion-pa.googleapis.com/v1/udmevents?key=" + self.googleSecKey, data = dumps(payload), headers=headers)
-            while (r.status_code>=400):
+            attempt = 0
+            headers = {'Content-Type': 'application/json'}
+            payload = {"events": [mapper(response)]}
+            r = request("POST", "https://malachiteingestion-pa.googleapis.com/v1/udmevents?key=" +
+                        self.googleSecKey, data=dumps(payload), headers=headers)
```
Comment (shawndwells) [inline EventStreams/src/chronicleclient.py:80]: To future proof the code, is the API endpoint something that should be parameterized? (likely in future PR, not to hold this one up)

File: Dockerfile (hunk 1 above)
Comment (redhatrises) [reply Dockerfile:1]: Hmm.... probably, how would you want to handle that for the different FIPS capable distros?

File: EventStreams/src/chronicleclient.py (hunk 2 above)
Comment (redhatrises) [reply EventStreams/src/chronicleclient.py:80]: Can you you open a ticket for this?

File: Dockerfile (hunk 1 above)
Comment (redhatrises) [reply Dockerfile:1]: Should a ticket be open to track it to fix separately from this PR?

Comment (redhatrises) [reply Dockerfile:1]: Opened https://github.com/CrowdStrike/Chronicle-Backstory-Integration/issues/5 to track this.

File: EventStreams/src/chronicleclient.py (hunk 2 above)
Comment (redhatrises) [reply EventStreams/src/chronicleclient.py:80]: Opened https://github.com/CrowdStrike/Chronicle-Backstory-Integration/issues/6 to track this.

**Review by shawndwells — APPROVED** [review]
Thanks for opening the other tracking tickets

**Pull request merged**